File: native/mixdog-graph/tests/fixtures/calls/kotlin/sample.kt
// hidden()
val quoted = "hidden()"

fun inner() {}
fun leaf(x: Int) = x
fun nest(x: Int) {}
fun helper() {}
fun seed() {}
fun plain() = 0

class Chain {
  fun b(): Chain = this
  fun c(): Chain = this
}

val g = plain()

fun run(a: Chain) {
  inner()
  nest(leaf(1))
  a.b().c()
}

class Widget {
  fun ping() {}
  fun act() {
    helper()
    this.ping()
  }
}

fun seedLine() {
  val mark = "μ"; seed()
}

val w = Widget()

File: native/mixdog-graph/tests/fixtures/calls/javascript/sample.js
// hidden()
const quoted = "hidden()";

function inner() {}
function leaf(x) { return x; }
function nest(x) { return x; }
function helper() {}
function seed() {}
function plain() {}

class Chain {
  b() { return this; }
  c() { return this; }
}

plain();

function run(a) {
  inner();
  nest(leaf(1));
  a.b().c();
}

class Widget {
  ping() {}
  act() {
    helper();
    this.ping();
  }
}

const mark = "μ"; seed();
new Widget();
